css/new-design/browser.css

```
@@ -34,6 +42,10 @@ html.private-mode [data-pagelet=page] .a8c37x1j.d2edcug0.sn7ne77z.bixrwtb6 {
 /* Conversation: name */
 html.private-mode [data-pagelet=page] .d2edcug0.hpfvmrgz.qv66sw1b.c1et5uql.gk29lw5a.a8c37x1j.keod5gw0.nxhoafnm.aigsh9s9.tia6h79c.fe6kdd0r.mau55g9w.c8b282yb.iv3no6db.a5q79mjw.g1cxx5fr.lrazzd5p.oo9gr5id.oqcyycmt {
 	filter: blur(5px);
+}
+ /* Sidebar: name  */
+html.private-mode [data-pagelet=page] .d2edcug0.hpfvmrgz.qv66sw1b.c1et5uql.oi732d6d.ik7dh3pa.ht8s03o8.a8c37x1j.keod5gw0.nxhoafnm.aigsh9s9.d9wwppkn.fe6kdd0r.mau55g9w.c8b282yb.iv3no6db.a5q79mjw.g1cxx5fr.lrazzd5p.oo9gr5id.oqcyycmt {
+	filter: blur(5px);
 }
 /* Message list: sender profile picture */
 html.private-mode [data-pagelet=page] .pfnyh3mw.aovydwv3.j83agx80.h676nmdw.oi9244e8.oygrvhab {
```

This is broken for the same reason as the sidebar name, classes sometimes differ slightly. This should just patch the problem with sender profile pictures sometimes not being blurred.

```suggestion
html.private-mode [role=main] .pfnyh3mw.aovydwv3.j83agx80 > span {
```